Wiki-style-guide.md

```diff
@@ -86,3 +112,13 @@
   > * **Exploration:** An interactive lesson on Oppia.
 
 * Include punctuation at the ends of sentences. Also include punctuation at the ends of phrases that are exceptions to the "Write in complete sentences" rule above. For example, notice the period at the end of the example above.
+
```

Comment:
One suggestion. Instead of all the stuff below, do you just want to point them to https://ecotrust-canada.github.io/markdown-toc/ and ask them to use it to generate the ToC?

Reply:
I added a link to the tool, which I agree is great! I kept the list of rules so that we have them documented even if the tool disappears/changes.